### PR TITLE
Prevent brew cleanup on CI

### PR DESCRIPTION
**Pull request by b-onc**
...and various other CI/repo improvements.
Preventing `brew cleanup` will save approx 1.5m per job

### CODE REVIEW

File: .github/workflows/beta.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+  workflow_dispatch:
```
Comment (VojtaStavik) [inline .github/workflows/beta.yml:7]: How does this work?

**Review by VojtaStavik — APPROVED** [review]

File: .github/workflows/beta.yml (hunk 1 above)
Comment (b-onc) [reply .github/workflows/beta.yml:7]: it'll put a button "Run action" on the actions tab 😂

Comment (VojtaStavik) [reply .github/workflows/beta.yml:7]: Nice ... how long has this been available? :D

Comment (b-onc) [reply .github/workflows/beta.yml:7]: A couple of months...

**Pull request merged**